ansible/roles/operator-pipeline/templates/openshift/tasks/get-manifest-list-digests.yml

```diff
@@ -0,0 +1,77 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: get-manifest-list-digests
+spec:
+  params:
+    - name: manifest_digests
+      description: |
+        Comma-separated list of manifest digests generated by the IIB builds, in the
+        format of registry.redhat.io/redhat/test-index@sha256:123
+    - name: environment
+      description: |
+        Which environment the pipeline is running in. Can be one of [dev, qa, stage, prod]
+    - name: pipeline_image
+  results:
+    - name: docker_references
+      description: Comma-separated list of indices
+    - name: manifest_list_digests
+      description: Comma-separated list of digests from registry manifest lists
+  workspaces:
+    - name: registry-credentials
+      description: Docker config for retrieving the bundle image
+  steps:
+    - name: podman-manifest-inspect
+      image: "$(params.pipeline_image)"
+      script: |
+        set -xe
+
+        # no-op for dev or qa since IIB doesn't run
+        # output dummy/test values for signing purposes
+        if [[ $(params.environment) == "dev" || $(params.environment) == "qa" ]]; then
```

Comment:
What if we declare `environment` params as an env variable?

Reply:
That should be easy to do, but why? The only benefit I see is that it'll be slightly shorter.

Reply:
Because we want to set the registry based on env in another way in the future, then it will be easy if we are using the env variable rather than referencing the params every time. WDYT?

Reply:
Updated.